examples/symbol/multisig.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable tsdoc/syntax */
+import { MultisigAccountModificationTransactionBodyBuilder, UnresolvedAddressDto } from 'catbuffer-typescript';
+import { Key, SymbolAddress, SymbolDeadline, SymbolKeyPair, SymbolNetwork } from '../../src/core';
+
+const multisigKeyPair = new SymbolKeyPair(Key.createFromHex('1C16D0C8804546EFB4B11584AFACB294DFABF0D41E8E345F1F74BB6CAD162066'));
+const cosignatoryKeyPairs = [
+    new SymbolKeyPair(Key.createFromHex('C73DF84178896A5D63F91467E606A44B06AF3892DB4F55FAF9524DA902E0FF7E')),
+    new SymbolKeyPair(Key.createFromHex('FC998941EDC8979252523885E4524598441577A9CEBCF5F766D194A11A1C317E')),
+    new SymbolKeyPair(Key.createFromHex('3AD4475F288A90654033EC4A4DE86F44ED3E898A384BE3488A2F02B95CC9C772')),
+];
+
+/**
+ * Build Symbol transaction using the factory and sign.
+ */
+const network = new SymbolNetwork('testnet', 0x98, '3B5E1FA6445653C971A50687E75E6D09FB30481055E3990C84B25E9222DC1155'); // Specify network
```

Comment:
Should the user load the network from the static list? Should we have a SymbolNetwork.Testnet static value?